src/app/components/Search/Search.jsx

```diff
@@ -124,7 +124,13 @@ class Search extends React.Component {
   }
 
   render() {
-    const inputError = this.state.inputError;
+    let inputError = '';
+
+    if (this.state.inputError === 'noKeyword') {
+      inputError = 'Please enter a search term.'
+    } else if (this.state.inputError === 'dateFilterError') {
```

Comment:
Hmmm I don't think this makes sense. You're adding an error message for valid dates in the search component that doesn't know anything about the dates. The error handling and error messaged is displayed in the filter popup, not in the omnisearch.

Reply:
One thing we can discuss over this PR is where to have the error message for incorrect date input when no js. If we want to put it as the same place as no keyword, we probably have to put the message here, but I am open to another suggestions. @ricardoom @seanredmond @wlla

Reply:
You can visually place it near the omnisearch, but, when we're talking about React components, it doesn't make sense to introduce the error for the date filters in the omnisearch component. So I'm against making this change. If you want to put the error near the omnisearch in a different component or under it, that's fine.

Reply:
As mentioned on stand up, I am open to whatever the place to have this error message, so let me know folks!